.github/data/dependencies.yml

```diff
@@ -1,28 +0,0 @@
-- name: dotnet-aspnetcore
```

Comment:
for some reason in the UI this file isn't showing up as fully deleted, just empty if you can delete it please!

Reply:
Oh yeah I just emptied it. I will delete it.